modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/editor/BalloonEditor.js

```diff
@@ -42,6 +42,15 @@ const BalloonEditor = ({config = {}, contents, name, ...otherProps}) => {
 		title: false,
 	};
 
+	const imageDirectionShim = editorConfig.toolbarImage.includes(
```

Comment:
I added this conditional here to prevent any side effects when not using ImageAlign* plugins

Reply:
With the less invasive change to `overflow: auto`, #1339 (comment), this kind of damage control is not needed.

Reply:
See https://github.com/liferay-frontend/liferay-portal/pull/1339/commits/df033efedceb335c9f7712859befaa2bb5003145